pom.xml

```diff
@@ -115,8 +115,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>matrix-project</artifactId>
-            <version>1.4</version>
-            <scope>test</scope>
+            <version>1.7</version>
+            <type>jar</type>
```

Comment:
Yes, it was before the split. We were using newer Matrix project just to fix integration bug, but the dependency was already there